### ISSUE TITLE
2021 season missing some roof info

### ISSUE THREAD

**Issue by greerreNFL** [nflverse/nfldata#17]
roof field appears to be null for a handful of games

a few examples...
2021_03_CAR_HOU
2021_05_NE_HOU
2021_12_TB_IND
2021_12_NYJ_HOU

Looks like it could be stadium specific

**Comment by guga31bb** [comment]
Yeah these are retractable roof stadiums where "open/closed" hasn't been filled in. Tagging @leesharpe since he knows more about this.

I recommend just replacing any NAs with "retractable" or something like this; this is how nflfastR works
